### ISSUE TITLE
enums with spaces throw an error

### ISSUE THREAD

**Issue by brookjordan** [acacode/swagger-typescript-api#71]
For example I have:
```yml
enum: [resolved, new, in progress]
```

And I get this error:
```shell
☄️  start generating your typescript api
SyntaxError: An enum member name must be followed by a ',', '=', or '}'. (19:6)
  17 |   resolved = "resolved",
  18 |   new = "new",
> 19 |   in progress = "in progress" 
     |      ^
  20 |  }
  21 | 
    at t (/project/node_modules/prettier/parser-typescript.js:1:287)
    at Object.parse (/project/node_modules/prettier/parser-typescript.js:14:2633274)
    at Object.parse (/project/node_modules/prettier/index.js:11370:19)
    at coreFormat (/project/node_modules/prettier/index.js:14784:25)
    at format (/project/node_modules/prettier/index.js:15019:75)
    at formatWithCursor (/project/node_modules/prettier/index.js:15035:12)
    at /project/node_modules/prettier/index.js:51620:12
    at Object.format (/project/node_modules/prettier/index.js:51640:12)
    at /project/node_modules/swagger-typescript-api/src/index.js:89:39 {
```

This is because it doesn’t add quotes around the enum declaration. That is, instead of outputting:
```ts
export enum Enum {
  resolved = "resolved",
  new = "new",
  in progress = "in progress",
}
```
should be:
```ts
export enum Enum {
  resolved = "resolved",
  new = "new",
  "in progress" = "in progress",
}
```

**Comment by js2me** [comment]
@brookjordan Thanks for this issue, it will be fixed in next release!

**Comment by js2me** [comment]
I think it should be better like:   
![image](https://user-images.githubusercontent.com/16340911/90562394-c1c7ad00-e1aa-11ea-9b41-5b3400802ccb.png)

**Comment by js2me** [comment]
Fixed in 1.12.0!
